scripts/upgrade

```diff
@@ -29,6 +29,35 @@ if [ -z "${secret_key:-}" ]; then
 	ynh_app_setting_set --app=$app --key=secret_key --value=$secret_key
 fi
 
+#=================================================
+# KEEP CONFIGS IF NEEDED
+#=================================================
+
+if [ -f "$install_dir/config.local.yunohost.php" ]
+then
+	ynh_script_progression --message="cleaning conf files..." --weight=5
+	mv "$install_dir/config.local.php" "$install_dir/config.local.php.old"
+	mv "$install_dir/config.local.yunohost.php" "$install_dir/config.local.php"
+	chmod 650 "$install_dir/config.local.php"
+	chown $app:$app "$install_dir/config.local.php"
+	ynh_store_file_checksum --file="$install_dir/config.local.php"
+	ynh_secure_remove "$install_dir/config.local.php.old"
+fi
+
+ynh_script_progression --message="removing old conf files if exists..." --weight=5
+
+if [ -f "$install_dir/config.local.user.php" ]; then
+	ynh_secure_remove "$install_dir/config.local.user.php"
```

Comment:
So if the file exist, we remove it, we don't move it to `$data_dir`?

Reply:
no, because the folder `$data_dir` or `data_dir/data` sometime didn't exist at this step and because we prefer use a clean new file for this upgrade...

Reply:
Also the path have changed, for example `DATA_ROOT`. If you are OK, then you can add an approving review...

Reply:
OK!